### DISCUSSION TITLE
UI much simpler since 0.1.4

### DISCUSSION THREAD

**Discussion by hoodie** [tokio-rs/console#400] [Q&A]
Hi there, just a quick question. I see the UI has been much simplified since 0.1.4, no more unicode characters, histogram is no longer rendered.

I was wondering whether this is a bug or not, since I didn't find a good explanation in the release notes or documentation.

thank you

**Comment by hawkw** [comment]
Hi, it sounds like you're hitting this bug: https://github.com/tokio-rs/console/issues/372, where the `ascii-only` setting was unintentionally flipped, so that the default configuration (with Unicode support enabled) actually disables the Unicode-based UI. That issue was fixed in #377, but I think I may have forgotten to publish a release with that change. I'll try to get one out soon!

In the meantime, you can try installing `tokio-console` from the `main` branch and see if your issues are resolved?

**Reply by hoodie** [reply]
that would explain why I got confused, because I was at one point trying out every single version since 0.1.3 to figure out what was going wrong
